docs/eventing/samples/kafka/channel/README.md

```diff
@@ -178,8 +181,8 @@ To use TLS authentication you must create:
 
 **NOTE:** Kafka channels require these files to be in `.pem` format. If your files are in a different format, you must convert them to `.pem`.
 
-
-1. Create the certificate files as secrets in your chosen namespace:
+For the consolidated channel type, create the certificate files as secret
```

Comment:
Procedures should remain as numbered steps

Reply:
I have replaced the numbering